app/scripts/image-transform.js

```diff
@@ -24,9 +24,45 @@ var imageTransform = (function(_) {
       }
       return images;
     },
+    image: function(data) {
```

Comment:
This function should be tested in client-config-image-test to ensure that the returned object (with all of its properties) is created correctly.